dhall-openapi/openapi-to-dhall/Main.hs

```diff
@@ -191,6 +210,16 @@ parseOptions = Options <$> parseSkip <*> parsePrefixMap' <*> fileArg <*> crdArg
         <> Options.Applicative.help "Specify prefix mappings as 'prefix1=importBase1,prefix2=importBase2,...'"
         <> Options.Applicative.metavar "MAPPING"
         )
+    parseSplits' =
+      option Data.Map.empty $ Options.Applicative.option parseSplits
+        (  Options.Applicative.long "splitPaths"
+        <> Options.Applicative.help
+          "Specifiy path and model name pairs with paths being delimited by '~' and pairs separated by '=' for which \ 
+          \definitions should be aritifically split with a ref: \n\
+          \'com.example.v1.Certificate~spec=com.example.v1.CertificateSpec'\n\
```

Comment:
Perhaps instead of `com.example.v1.Certificate~spec` we could use something like `(com.example.v1.Certificate).spec`, that way we can keep the `.` syntax for the path

Reply:
I like that idea.